pkg/pipelines/pipelines.go

```diff
@@ -398,3 +400,62 @@ func GetLatestPipelinerun(c *clients.Clients, namespace string) (string, error)
 	return prs.Items[0].Name, nil
 
 }
+
+func CheckLogVersion(c *clients.Clients, binary, namespace string) {
+	prname, err := GetLatestPipelinerun(store.Clients(), store.Namespace())
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun: %v", err))
+		return
+	}
+
+	switch binary {
+	case "tkn-pac":
+		expectedVersion := os.Getenv("PAC_VERSION")
+		// Get PipelineRun logs
+		logsBuffer, err := GetPipelinerunLogs(c, prname, namespace)
+		if err != nil {
+			testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun logs: %v", err))
+			return
+		}
+		// Convert logs to string and extract tkn-pac version using regex
+		logs := logsBuffer.String()
```

Comment:
This code is duplicated in both branches, refactor it out of switch-case. Additionally, you can add `.String()` directly to the function call

Reply:
Moved log fetching out of switch case